test/functional/rpc_fundrawtransaction.py

```diff
@@ -1073,6 +1074,81 @@ def test_external_inputs(self):
 
         self.nodes[2].unloadwallet("extfund")
 
+    def test_add_inputs_default_value(self):
+        self.log.info("Test 'add_inputs' default value")
+
+        # Create and fund the wallet with 5 BTC
+        self.nodes[2].createwallet("test_preset_inputs")
+        wallet = self.nodes[2].get_wallet_rpc("test_preset_inputs")
+        addr1 = wallet.getnewaddress(address_type="bech32")
+        self.nodes[0].sendtoaddress(addr1, 5)
+        self.generate(self.nodes[0], 1)
+
+        # Covered cases:
+        # 1. Default add_inputs value with no pre-set inputs (add_inputs=true):
+        #       Expect: automatically add coins from the wallet to the tx.
+        # 2. Default add_inputs value with preset inputs (add_inputs=false):
+        #       Expect: disallow automatic coin selection.
+        # 3. Explicit add_inputs=true:
+        #       Expect: include inputs from the wallet.
```

Comment:
In 1350349f5951a4e1fdc7d92018cb5be6c014e99a "test: add coverage for 'add_inputs' dynamic default value"

I think it would be useful to have a case for having both add_inputs=true and preset inputs.

Reply:
done